Dockerfile.gitlab

```diff
@@ -77,3 +84,6 @@ ENV PATH $PATH:$GRADLE_HOME/bin
 ENV PATH $PATH:$ANDROID_HOME/platform-tools
 ENV PATH $PATH:$ANDROID_SDK_ROOT/build-tools/${ANDROID_BUILD_TOOLS}:$ANDROID_NDK
 
+# Woke
+RUN curl -sSfL https://git.io/getwoke | bash -s -- -b /usr/local/bin
+ENV PATH $PATH:/usr/local/bin
```

Comment:
nit: missing carriage return at end of file

Reply:
Good point thx :)